errtracker/errtracker.go

```diff
@@ -91,6 +91,13 @@ func snapConfineProfileDigest(suffix string) string {
 	return fmt.Sprintf("%x", md5.Sum(profileText))
 }
 
+func detectReExec() string {
```

Comment:
not quite sure why the helper?  wouldn't just returning true|false out of osutil.GetenvBool("SNAP_DID_REEXEC") be enough and change  DidReExec => DidSnapdReExec ?